Src/Newtonsoft.Json/Linq/JTokenWriter.cs

```diff
@@ -88,15 +88,18 @@ public JTokenWriter()
         }
 
         /// <summary>
-        /// Flushes whatever is in the buffer to the underlying streams and also flushes the underlying stream.
+        /// Flushes whatever is in the buffer to the underlying <see cref="JContainer"/>.
```

Comment:
While it seemed safe to remove the statement about the underlying stream being flushed (since a `JContainer` *cannot* be flushed) I wasn't so sure about the first half of the sentence; you might introduce buffering for `JTokenWriter` in the future, so I left this as is.